Cargo.toml

```diff
@@ -30,19 +30,18 @@ fontdb = "0.9"
 if_chain = "1"
 image = { version = "0.24", default-features = false, features = ["png", "jpeg", "gif"] }
 log = "0.4"
-miniz_oxide = "0.5"
+miniz_oxide = "0.7"
 once_cell = "1"
 pdf-writer = "0.6"
 pixglyph = "0.1"
 regex = "1"
 resvg = { version = "0.22", default-features = false }
 roxmltree = "0.14"
-rustybuzz = "0.5"
```

Comment:
I'd like to wait with bumping rustybuzz until we have bumped `usvg`. Otherwise, it will be in the binary twice. Same with `kurbo`.